docs/content/en/rules/provider_known_vulnerability.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "CI Component with a Known Vulnerability used"
+slug: known_vulnerability
+url: /rules/known_vulnerability/
+rule: known_vulnerability
+severity: warning
+---
```

Comment:
Hmm that's copy pasted from existing `known_vulnerability` 

Should we just use the existing rule and add extra documentation in it.

If we make a new one, then the previous rule ID is too generic .... still time to break the rule ID I think we're below `v1.x`... think about it

Let's say we rename `known_vulnerability` to `known_vulnerability_in_build_component` and the new one is `known_vulnerability_in_build_platform` or `..._in_build_environment`